examples/Microsoft.DocumentDB_mongoClusters@2025-09-01/main.tf

```diff
@@ -20,6 +23,28 @@ variable "location" {
   default = "westus3"
 }
 
+variable "replica_location" {
+  type    = string
+  default = "centralus"
+}
+
+variable "admin_username" {
+  type    = string
+  default = "mongoAdmin"
+}
+
+variable "admin_password" {
+  type      = string
+  default   = "terraformTest@2025!"
```

Comment:
Please remove the default values for sensitive variables and add a description.

Reply:
fixed